cmd/api.go

```diff
@@ -48,7 +47,7 @@ func NewAPICmd(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("could not initialise Prometheus exporter: %w", err)
 			}
 
-			k8sConfig, err := clientcmd.BuildConfigFromFlags(opts.masterURL, opts.kubeConfig)
+			k8sConfig, err := buildKubeClientConfig(cfg.MasterURL, opts.kubeConfig, cfg.KubeClientTimeout)
```

Comment:
More for consistency but why is `kubeConfig` private and are `MasterURL` and `KubeClientTimeout` not?

Reply:
The `opts` are not used anywhere else outside the `cmd` package.
We can move it to a separate package to make them all public and consistent but that seems like more trouble than it's worth.

Reply:
I think we have a misunderstanding here, let my try a different angle.
Could you explain why you made `MasterURL` public? Can you also explain why `KubeClientTimeout` was created as public.

Reply:
> Could you explain why you made `MasterURL` public?

Seems like I got confused here (and changed `opts.masterURL` to `cfg.MasterURL` since they were already inconsistent([api], [controller], and [proxy]) in the first place. I reverted it to use `opts.masterURL` again in 5e7ef06.  
This and several other items from config can be moved to `opts`, but I think it should be done in another pull-request.

> Can you also explain why `KubeClientTimeout` was created as public.

For consistency; both `controller.Config` and `proxy.Config` are where envconfigs are located. I suppose we can add/create `cmd.Config` and move it there, but it feels too much for one envconfig.

[api]: https://github.com/hellofresh/kangal/blob/ee235493c32b20adeded3cf69d678e9525903d70/cmd/api.go#L51
[controller]: https://github.com/hellofresh/kangal/blob/ee235493c32b20adeded3cf69d678e9525903d70/cmd/controller.go#L57
[proxy]: https://github.com/hellofresh/kangal/blob/ee235493c32b20adeded3cf69d678e9525903d70/cmd/proxy.go#L51